Resources/config/events_deferring.yml

```diff
@@ -3,6 +3,11 @@ parameters:
   event_dispatcher.class: Smartbox\Integration\FrameworkBundle\Tools\EventsDeferring\EventDispatcher
 
 services:
+  event_dispatcher:
```

Comment:
We need to validate that this is going to work in Symfony 2.8, that based on the documentation it should https://symfony.com/doc/2.8/service_container/service_decoration.html